docs/en-US/installation.md

```diff
@@ -36,7 +36,7 @@ You can also remove the `--provider=hyperv` part by modifying `config/custom.yml
 
 ```yaml
 vm_config:
-  memory: 2048
+  memory: 100TB
```

Comment:
Unless a machine has 100TB of ram this will fail, I've never encountered such a machine so this is not a good default. If this was an upper limit of an expandable RAM that would be different but that's not how this works.

If you ask for more RAM than there is physical RAM in the machine things will break